### ISSUE TITLE
[Bug] Manifest version showing incorrectly on docs site

### ISSUE THREAD

**Issue by jelstongreen** [dbt-labs/dbt-core#5381]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

https://docs.getdbt.com/reference/artifacts/manifest-json - Current schema shows as v4 for version 1.1

### Expected Behavior

Should show v5

### Steps To Reproduce

Go to https://docs.getdbt.com/reference/artifacts/manifest-json

### Environment

```markdown
- OS: N/A
- Python: N/A
- dbt: N/A
```

### What database are you using dbt with?

other (mention it in "Additional Context")

### Additional Context

N/A

**Comment by jtcohen6** [comment]
@jelstongreen You're right! Thanks for catching. I'm going to move this issue over to the docs.getdbt.com repo, since that's where the change is needed
